code/modules/food_and_drinks/food/snacks_pastry.dm

```diff
@@ -48,18 +54,12 @@
 	. = ..()
 	extra_reagent = pick("nutriment", "capsaicin", "frostoil", "krokodil", "plasma", "cocoa", "slimejelly", "banana", "berryjuice", "omnizine")
```

Comment:
to note, both on live and in this PR, `"banana"` / banana juice reacts with the sugar in the donut and causes it to consume 2 sugar, 2 banana juice, and creates 20u of Laughter instead.